app/src/main/java/com/github/mik629/android/fundamentals/data/db/models/ActorDbEntity.kt

```diff
@@ -0,0 +1,36 @@
+package com.github.mik629.android.fundamentals.data.db.models
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.github.mik629.android.fundamentals.data.db.models.ActorDbEntity.Companion.ACTORS_TABLE_NAME
+import com.github.mik629.android.fundamentals.domain.model.Actor
+
+@Entity(tableName = ACTORS_TABLE_NAME)
+data class ActorDbEntity(
+    @PrimaryKey
+    @ColumnInfo(name = COLUMN_NAME_ACTOR_ID)
+    val actorId: String,
+    val name: String,
+    val photoUrl: String?
+) {
+    companion object {
+        const val ACTORS_TABLE_NAME = "actors"
+
+        const val COLUMN_NAME_ACTOR_ID = "actorId"
+    }
+}
+
+fun toActor(entity: ActorDbEntity) =
```

Comment:
You mean make `toActor` as extension but leave `fromActor` as is?           
I understand that using extensions is better because it's just there and you don't need to look for a static convertor method somewhere in code base. I just wanted `to` and `from` routines behave the same.

Reply:
both should be extensions
just think about preferred naming